src/number_theory/arithmetic_function.lean

```diff
@@ -623,16 +643,23 @@ begin
   simp [pow, (ne_of_lt (nat.succ_pos k)).symm],
 end
 
+lemma pow_zero_eq_zeta : pow 0 = ζ := by { ext n, simp }
+
 /-- `σ k n` is the sum of the `k`th powers of the divisors of `n` -/
 def sigma (k : ℕ) : arithmetic_function ℕ :=
 ⟨λ n, ∑ d in divisors n, d ^ k, by simp⟩
 
 localized "notation `σ` := nat.arithmetic_function.sigma" in arithmetic_function
 
-@[simp]
 lemma sigma_apply {k n : ℕ} : σ k n = ∑ d in divisors n, d ^ k := rfl
 
-lemma sigma_one_apply {n : ℕ} : σ 1 n = ∑ d in divisors n, d := by simp
+lemma sigma_one_apply {n : ℕ} : σ 1 n = ∑ d in divisors n, d := by simp [sigma_apply]
```

Comment:
Maybe `n` should be explicit here and in the following lemmas?

Reply:
Good idea, thanks!